inc/saiport.h

```diff
@@ -586,6 +586,22 @@ typedef enum _sai_port_path_tracing_timestamp_type_t
 
 } sai_port_path_tracing_timestamp_type_t;
 
+/**
+ * @brief Attribute data for #SAI_PORT_ATTR_UNRELIABLE_LOSS_OF_SIGNAL
+ */
+typedef enum _sai_port_unreliable_loss_of_signal_t
+{
+    /**  Unreliable loss of signal is not enabled for transceiver */
+    SAI_PORT_UNRELIABLE_LOSS_OF_SIGNAL_DISABLE,
+
+    /**  Unreliable loss of signal is enabled for transceiver */
+    SAI_PORT_UNRELIABLE_LOSS_OF_SIGNAL_ENABLE,
+
+    /**  Unreliable loss of signal is enabled only for interested transceiver */
```

Comment:
@JaiOCP what does interested transceiver mean? Who decides when to select auto and how?